doc/02-Installation.md

```diff
@@ -202,6 +203,80 @@ After creating the database, you can import the Icinga DB schema using the follo
 mysql -u root -p icingadb </usr/share/icingadb/schema/mysql/schema.sql
 ```
 
+### Configuring PostgreSQL <a id="configuring-icingadb-pgsql"></a>
+
+#### Installing PostgreSQL database server <a id="installing-database-pgsql-server"></a>
```

Comment:
Please remove this chapter. I will also remove the MySQL installation instructions in a separate PR.

Reply:
You can do it if you want.

Reply:
Btw.: why?

Reply:
There are so many tutorials on how to install a database that it makes no sense to explain it in each of our products that require a database.

Reply:
"in each of our products"... OK, this is definitely true. But IMAO we should at least provide a link to a tutorial. Maybe you already heard of that discussion, maybe not, so I'll tell you (again):

@gunnarbeutner doubted the need of a such chapter.
@dnsmichi : "I won’t explain to anyone how to setup a database."

To me @dnsmichi seemed to expect a lot of questions. Actually the same question.

I have no problem with not providing even a link, but... I won’t explain to anyone how to setup a database. :P